### DISCUSSION TITLE
aligning ZNP and Bellows configuration

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1011] [Q&A]
I'm trying to align the radio configuration. I have found that ZNP always try to identify UNKNOW devices, and I thing Z-Stack firmware enable Source Routing.

On Bellows, I have found that Source Routing and  "handle_unknown_devices" are by default set to True.

Is my understanding correct that if on Bellows, I set

"handle_unknown_devices": True
"source_routing": True

Is there any other configuration I should have a look ?

Is source_routing impacting only when forming the network, or will it also be efficient when on an existing network ?

**Comment by Hedda** [comment]
Did you ever get this question answered about these configurations and inconsistencies between different radio libraries zigpy?

And does the same thing happen if using zigpy-cli when forming a network?
